x-pack/test/ingest_manager_api_integration/apis/file.ts

```diff
@@ -47,7 +47,7 @@ export default function ({ getService }: FtrProviderContext) {
             '/api/ingest_manager/epm/packages/filetest/0.1.0/kibana/visualization/sample_visualization.json'
           )
           .set('kbn-xsrf', 'xxx')
-          .expect('Content-Type', 'application/json; charset=utf-8')
+          .expect('Content-Type', 'text/plain; charset=utf-8')
```

Comment:
These changes will go away once this PR is merged: https://github.com/elastic/kibana/pull/70716